changelogs/fragments/70056-add-a-param-to-apt-module-to-avoid-unintended-uninstalls.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - apt - add 'no_remove' param to apt module to avoid unintended package removals (https://github.com/ansible/ansible/issues/63231)
```

Comment:
```suggestion
  - apt - add ``fail_on_autoremove`` param to apt module to avoid unintended package removals (https://github.com/ansible/ansible/issues/63231)
```